src/index.ts

```diff
@@ -43,15 +43,24 @@ class AbortError extends Error {
   }
 }
 
+interface ResourceLimits {
+  maxOldGenerationSizeMb? : number,
+  maxYoungGenerationSizeMb? : number,
+  codeRangeSizeMb? : number
+}
+
 interface Options {
   // Probably also support URL here
   filename? : string | null,
   minThreads? : number,
   maxThreads? : number,
   idleTimeout? : number,
   maxQueue? : number,
-  concurrentTasksPerWorker? : number
-  useAtomics? : boolean
+  concurrentTasksPerWorker? : number,
+  useAtomics? : boolean,
+  maxOldGenerationSizeMb? : number,
+  maxYoungGenerationSizeMb? : number,
+  codeRangeSizeMb? : number,
 }
```

Comment:
If you really do want to list these individually, you might want to use `interface Options extends ResourceLimits` instead